apps/react-app/src/core/routes/routes.ts

```diff
@@ -0,0 +1,11 @@
+interface SwitchRoutes {
+  root: string;
+  scene1: string;
+  scene2: string;
```

Comment:
Let's change this to the real scenario, we got:

- Home page (paste json)
- Choose template + export


We can name it:

homeScene
TemplateScene